### DISCUSSION TITLE
ARM won't auto-start ripping

### DISCUSSION THREAD

**Discussion by dongafford** [automatic-ripping-machine/automatic-ripping-machine#653] [Q&A]
I've been through the FAQ and everything looks good except the following:

When I tail syslog I'm seeing the following entry when inserting or ejecting a disk:
systemd-udevd[2183]: sr0: Process '/opt/arm/scripts/thickclient/arm_wrapper.sh sr0' failed with exit code 127.

I'm on my third complete rebuild at this point. I've gone through installing by script and manually installing with commands from the FAQ. Neither worked. I was finally able to get things installed by following the steps in the **Working install guide 14 Nov 2022** discussion.

Here's my lsb-release output.
-------------------------------------------
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.5 LTS"
-------------------------------------------

I'm running Ubuntu Server. 

Everything else is working. I can run the "sudo -u arm /usr/bin/python3 /opt/arm/arm/ripper/main.py -d sr0" command and everything works, it's just the auto-start. 

My 51-automedia.rules is stock at this point. 

If I run the command manually (/opt/arm/scripts/thickclient/arm_wrapper.sh sr0), I get the following:
ID_CDROM_MEDIA_BD: command not found. 

This led me to comment out that line in the file and it then moved the error to ID_CDROM_MEDIA_DVD: command not found. After I commented out all three lines, the application will start to automatically rip the DVD. 

#ID_CDROM_MEDIA_BD = Blu-ray
#ID_CDROM_MEDIA_CD = CD
#ID_CDROM_MEDIA_DVD = DVD

Are these supposed to be commented out? Am I bypassing some important part of the application by doing so? If so, am I missing a PIP or APT package? Is this a bug?

Thanks in advance for any help.

**Comment by shitwolfymakes** [comment]
Sorry to hear about the trouble you've been having. Have you tried using the docker container?

**Reply by 1337-server** [reply]
Error code 127 means that one of the programs cannot be found.

Make sure that both HandbrakeCLI and makemkvcon is installed and can be run via the terminal.

I suggest manually going through each dependency and make it sure it's all installed correctly.

**Reply by dongafford** [reply]
I haven't tried the docker container as I am pretty new to Docker and have had issues connecting drives with other applications. I've even seen bugs to that effect (connecting USB drives) on this sub. I'll go that way if I have to, but it seems like things are working now. 

It looks like the lines I commented out in my arm_wrapper.sh file are commented out in the version on GitHub. Based on this, I will assume that it is safe to comment them out and will leave them like this. 

The programs that could not be found were the uncommented ID_CDROM_MEDIA lines in the arm_wrapper.sh. Somehow the install had a version of the file where those lines were uncommented and read as commands. Once I commented them out, it worked like it should.
